src/test/java/uk/gov/pay/api/it/PaymentsResourceGetIT.java

```diff
@@ -765,6 +778,21 @@ public void getPaymentWithNoAuthorisationSummaryThroughConnector() {
         response.body("authorisation_summary", is(nullValue()));
     }
 
+    @Test
+    public void getPaymentShouldNotIncludeNoAuthorisationSummaryThroughConnectorWhenThreeDSecureRequiredIsFalse() {
```

Comment:
The double negative `ShouldNotIncludeNo` doesn’t look right.

Reply:
But I like double negatives :(